src/app/public/features/components/header/header.component.ts

```diff
@@ -1,12 +1,21 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Input } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.css'],
 })
-export class HeaderComponent implements OnInit {
-  constructor() {}
+export class HeaderComponent {
+  @Input() searchbox: any = '';
```

Comment:
the type for this var should be 'string', no?

Reply:
yes, changed here in [PR WPA-button](https://github.com/MariaAlba1306/github/pull/4)